xwiki-platform-core/xwiki-platform-wysiwyg/xwiki-platform-wysiwyg-api/src/main/java/org/xwiki/wysiwyg/script/WysiwygEditorScriptService.java

```diff
@@ -121,6 +121,14 @@ public boolean isSyntaxSupported(String syntaxId)
         }
     }
 
+    /**
+     * @return The syntax identifier of the rendered HTML.
```

Comment:
Missing ``@since``.

Reply:
I've added `@since` and also `@Unstable` as this is a new public API (at least that's my understanding).